### PR TITLE
histogram: Fix for bug #10705 "Dragging histogram yields crazy results"

### PR DESCRIPTION
**Pull request by avsmal**
Mouse scrolling and dragging on the right part of the histogram (exposure) was implemented in terms of "white", not in terms of "exposure". That caused incorrect behavior: the slightest drag in any direction changed exposure by several EV, scrolling changed exposure in non-linear way.

### CODE REVIEW

**Comment by avsmal** [comment]
I'm not sure whether it is a good idea to duplicate definition of white2exposure and exposure2white from iop/exposure.c to libs/histogram.c, but I don't know what header file is suitable for these two macroses.

**Comment by houz** [comment]
Maybe you could instead change the `set_white` and `get_white` hooks to `set_exposure` and `get_exposure` so you keep the calculations in exposure.c.

**Comment by LebedevRI** [comment]
@avsmal yes, please try modifying `set_white` and `get_white` hooks.
Also, while there, it should make it easy to fix https://redmine.darktable.org/issues/10073

**Comment by avsmal** [comment]
@houz @LebedevRI OK, I'll see what I can do.

**Comment by avsmal** [comment]
@houz I changed hooks to keep all calculations in exposure.c. #10073 is also fixed. @LebedevRI

File: src/develop/develop.h [hunk 1]
```diff
@@ -244,7 +244,7 @@ void dt_dev_configure(dt_develop_t *dev, int wd, int ht);
 void dt_dev_invalidate_from_gui(dt_develop_t *dev);
 
 /*
- * exposure plugin hook, set the white level
+ * exposure plugin hook, set the exposure level and the black level
```
Comment (LebedevRI) [inline src/develop/develop.h]: Here and next:
`exposure level` is kinda meaningless to me.
Maybe either just `exposure` or `exposure compensation`

**Comment by avsmal** [comment]
@LebedevRI that's true. I've changed to "exposure".

**Comment by LebedevRI** [comment]
Okay, looks about right.
I will increase step of exposure change when scrolling histogram though.

**Comment by avsmal** [comment]
@LebedevRI is it possible to merge it into darktable-2.0.x?

**Comment by LebedevRI** [comment]
@avsmal yeah, i *think* it is not too intrusive. i wanted to wait some time, and so far no complaints.
So i will look into cherry-picking it before 2.0.3

**Pull request merged**